Makefile

```diff
@@ -65,6 +65,8 @@ proto:
 	rm -rf www/grpc/gen
 	$(SED_CMD) 's/{{ VERSION }}/$(VERSION)/g' www/grpc/buf/openapi.config.yaml
```

Comment:
I am not sure why it is still here. Please remove any VERSION from Makefile.
Let's keep Makefile simple

Reply:
VERSION is here not bad.